docker_build/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# An integration test & dev container which builds and installs cuDF from main
+ARG CUDA_VERSION=10.2
```

Comment:
Can we bump to CUDA 11.2 just to make sure that works as expected?

Reply:
FYI post CUDA 11.1 the CUDA images in `nvidia/cuda` now require the use of the patch version. So this would need to be `11.2.2` for it to work

Reply:
Done 👍